app/controllers/terms_controller.rb

```diff
@@ -35,4 +36,11 @@ def render_404
   def vocabulary
     @vocabulary ||= Vocabulary.find(params[:vocabulary_id])
   end
+
+  def authorize
```

Comment:
sure; since I don't know rails very well yet, is there a  filenaming convention? does it stay in the controllers directory?